master.yml

```diff
@@ -3,6 +3,7 @@
 - import_playbook: playbooks/deactivate_password_playbook.yml
 - import_playbook: playbooks/enable_camera_playbook.yml
 - import_playbook: playbooks/common_playbook.yml
+- import_playbook: install_vision_dependencies.yml
```

Comment:
I believe, it should be `import_playbook: playbooks/install_vision_dependencies.yml`